app/static/voice.js

```diff
@@ -0,0 +1,67 @@
+window.addEventListener("load",(event) => {
+ 
+  let synth = window.speechSynthesis;
+  let voices = [];
+  let voiceSelect = document.querySelector("#voice");
+
+  
+  function listVoices(){
+    voices = synth.getVoices();
+    let selectedIndex = voiceSelect.selectedIndex < 0 ? 0 : voiceSelect.selectedIndex;
+    
+    voiceSelect.innerHTML = '';
+    for(let voice of voices) {
+      var option = document.createElement('option');
```

Comment:
Prefer `let` or `const`

Reply:
thanks :)
Done for this and other declarations